docs/learn/bitesize-spaceshooter.md

```diff
@@ -0,0 +1,83 @@
+---
```

Comment:
Just to get some feedback on the level of scope we want here. Should I have gone into more detail? I tried to keep this very brief as more like an short introductory text which you would read before digging into the sample. Is that what we are trying to go for here?

Reply:
I used the content from the Space Shooter RFD. If this will have additional deeper content, going further into the code project, I can save this for future additions, or can add more content if you would like to expand. When the code repo goes public, this content could be helpful for users. 

As to scope, intro to the code, what features it highlights, how to use the module?